docs/contributing/how-to-make-a-reproducible-test-case.md

```diff
@@ -15,22 +15,48 @@ The [most important part of any bug report](https://developer.mozilla.org/en-US/
-- Add any Gatsby plugins that relate to the issue. For example, if you're having problems with Gatsby MDX you should install and configure [`gatsby-plugin-mdx`](/plugins/gatsby-plugin-mdx/). Only add plugins that are needed to demonstrate the problem.
+- Install Gatsby CLI if you haven’t using this [guide](https://www.gatsbyjs.com/docs/reference/gatsby-cli/).
+- Create a new Gatsby site with a starter, the official `gatsby-starter-minimal` starter is a great 'barebones' starting point here: `gatsby new bug-repro https://github.com/gatsbyjs/gatsby-starter-minimal` There are other Gatsby starters you can use [here](https://www.gatsbyjs.com/starters/)
+- Add any Gatsby plugins that relate to the issue. For example, if you're having problems with Gatsby MDX you should install and configure [`gatsby-plugin-mdx`](/plugins/gatsby-plugin-mdx/) in the directory of your site (i.e bug-repro). Remember to update your `gatsby-config.js` with the plugin you installed and only add plugins that are needed to demonstrate the problem.
 - Add the code needed to recreate the error you've seen.
-- Publish the code (your GitHub account is a good place to do this) and then link to it when [creating an issue](/contributing/how-to-file-an-issue/).
-
-## Online Development Environments
-
-Instead of cloning `gatsby-starter-minimal` to your local computer and working on it there, you can also use Online IDEs that set up the environment for you. Available options:
-
-- [CodeSandbox](https://githubbox.com/gatsbyjs/gatsby-starter-minimal)
-- [StackBlitz](https://githubblitz.com/gatsbyjs/gatsby-starter-minimal)
+- Build the site locally using `gatsby develop` and confirm you’re not seeing any errors.
+- Publish the code (your GitHub account is a good place to do this)
+- Use the repo to build a site on Gatsby cloud and look out for any build errors.
+- Share the link to the repo and build logs when reaching out to support. You can also share the repo if you’re [creating an issue](https://www.gatsbyjs.com/contributing/how-to-file-an-issue/)
 
 ## Benefits of reproducible test cases
 
 - Smaller surface area: By removing everything but the error, you don't have to dig to find the bug.
 - No need to publish secret code: You might not be able to publish your main site (for many reasons). Remaking a small part of it as a reproducible test case allows you to publicly demonstrate a problem without exposing any secret code.
 - Proof of the bug: Sometimes a bug is caused by some combination of settings on your machine. A reproducible test case allows contributors to pull down your build and test it on their machines as well. This helps verify and narrow down the cause of a problem.
 - Get help with fixing your bug: If someone else can reproduce your problem, they often have a good chance of fixing the problem. It's almost impossible to fix a bug without first being able to reproduce it.
+
+## **What if I’m unable to create a reproducible test case?**
```

Comment:
There's no need for making the headings bold

Reply:
Removed the heading